### PR TITLE
Add Syslog as logging src and refector old code

### PR DESCRIPTION
**Pull request by toxix**

### CODE REVIEW

File: templates/monitor.php [hunk 1]
```diff
@@ -101,6 +102,13 @@ function is_access_log_expired($logfile, $logExpiryInterval)
     return $lastLogEntry < $expiryThreshold;
 }
 
+function is_sys_log_expired($unit, $logExpiryInterval)
+{
+        $unitRef = escapeshellarg($unit);
+        $line = `journalctl -u $unit -n 1 -o cat`;
```
Comment (homecoded) [inline templates/monitor.php]: Alignment :D

File: README.md [hunk 2]
```diff
@@ -48,7 +48,14 @@ it is also possible to parse log file manually and feed it in the script:
 grep nginx /var/log/syslog | grep -v "health_check" | tail -1  | php monitor.php --interval 20minutes --termination "/sbin/shutdown -h now"
 ```
 
-PLEASE NOTE: only the last line of STDIN will be processed in this case, `--logfile` is ignored in this mode.
+PLEASE NOTE: only the last line of STDIN will only be processed if no other input is given by `--logfile` or `--syslog`
```
Comment (homecoded) [inline README.md]: 2 onlys! 1 only to many

File: templates/monitor.php (hunk 1 above)
Comment (toxix) [reply templates/monitor.php]: outdated, check the final PR.

File: README.md (hunk 2 above)
Comment (toxix) [reply README.md]: updated.

**Review by homecoded — APPROVED** [review]
All cool

**Comment by aboritskiy** [comment]
this is not working as expected, I'm getting dozens of those emails
```
Journal file /var/log/journal/4348e6a9fe504faa9cea37e7f3d48f6f/user-1003.journal is truncated, ignoring file.
```
cause relying on files is not working reliably when log rotate is in place.
Exactly for that reason the support for stdin was implemented before, we should rather use syslog as the service and feed the data via stdin rather than trying to read syslog files, cause those are rotated and chances that we will be reading empty file are quite high.

**Comment by toxix** [comment]
Hey @aboritskiy ,

We actualy are using the journalctl system service, not accessing any sys log files directly.

This looks more like a bug in journalctl or a corrupted syslog/journel.

see also https://github.com/sitewards/ansible-role-server-suicide/pull/4/files#diff-b460deece1743eaf5447d736027fa9dcR154

**Pull request merged**